---
layout: post
title: "Install Anvio"
author: "Andrea Telatin"
tags: [anvio]
image: anvio.jpg
---


## Anvi'o workshop requisites

> If you cannot have a working Anvi'o installation,
> please join as auditor, we can pair you with other participants!

### 1. Install Anvi'o

1.	On Windows,  you will need the Windows Subsystem for Linux (WSL) installed
a.	https://learn.microsoft.com/en-us/windows/wsl/install 
2.	Install Miniconda (on Windows, install the Linux version inside WSL!)
a.	https://docs.anaconda.com/free/miniconda/#quick-command-line-install	
b.	Ensure to install the latest stable vesion (v8) and not the development version
3.	Follow the instructions on Anvi’o website to install https://anvio.org/install
4.	Test the installation typing:
anvi-self-test --suite mini


### 2. Install the databases

5.	Download some extra databases, typing these commands from the command line (from the anvio-8 environment)
a.	Run `anvi-setup-scg-taxonomy`, to setup SCG taxonomy data using GTDB genomes.
b.	Run `anvi-setup-ncbi-cogs`, to setup NCBI’s COG database for quick annotation of genes with functions,
c.	Run `anvi-setup-kegg-data`, so anvi-estimate-metabolism and anvi-reaction-network find the database of KEGG orthologs ready when you need it.
