---
layout: post
title: "Install Anvio"
author: "Andrea Telatin"
tags: [anvio]
image: anvio.jpg
---


## Anvi'o workshop requisites

> If you cannot have a working Anvi'o installation,
> please join as auditor, we can pair you with other participants!

### 1. Install Anvi'o

> In general, follow [Anvio's instructions](https://anvio.org/install)

1.	On Windows,  you will need the **Windows Subsystem for Linux** (WSL) installed (and possibly the permission to install software from Computing, if you do not already have it).
    * See [instructions](https://learn.microsoft.com/en-us/windows/wsl/install)
2.	Install [Miniconda](https://docs.anaconda.com/free/miniconda/#quick-command-line-install) 
    * on Windows, install the Linux version inside WSL!
    * on Mac, you  might need to install XCode and other prerequisites, [see here](https://mmbdtp.github.io/modules/unix/Installing_programs_and_handling_Conda_environments/)
    * Ensure to install the **latest stable version (v8)** and not the development version
3.	Follow the instructions on Anvi’o website to install https://anvio.org/install
4.	Test the installation typing:
```bash
conda activate anvio-8
anvi-self-test --suite mini
```

### 2. Install the databases

5.	Download some extra databases, typing these commands from the command line (from the anvio-8 environment)
    * Run `anvi-setup-scg-taxonomy`, to setup SCG taxonomy data using GTDB genomes.
    * Run `anvi-setup-ncbi-cogs`, to setup NCBI’s COG database for quick annotation of genes with functions,
    * Run `anvi-setup-kegg-data`, so anvi-estimate-metabolism and anvi-reaction-network find the database of KEGG orthologs ready when you need it.

