---
layout: post
title: "Large Scale Analysis of Microbial Genomes"
author: "Leonardo De Oliveira Martins"
categories: events
tags: [events,large-scale]
image: large.jpg
---

## What
This workshop will discuss state-of-the-art methods in the large-scale analysis of microbial genomes, with a focus on protein clustering, orthology inference and structural phylogenetics.
We will discuss the latest advances in the field, including the use of artificial intelligence to improve the accuracy and scalability of these methods.

## Who
The two international groups invited for this workshop are [Christophe Dessimoz](https://orcid.org/0000-0002-2170-853X) & [Natasha Glover](https://orcid.org/0000-0003-1811-4340) 
from the University of Lausanne and Swiss Institute of Bioinformatics (SIB), and [Martin Steinegger](https://orcid.org/0000-0001-8781-9753) from Seoul National University. 
Both teams have collaborative work in the application of artificial intelligence to advance structural phylogenetics. 

Martin was involved in the development of popular microbial genomics tools like [mmseqs2](https://github.com/soedinglab/MMseqs2) and [PLASS](https://github.com/soedinglab/plass). 
He furthermore has been working with AlphaFold for clustering and classification of all known protein sequences based on their structure.
The focus of Christophe’s and Natasha’s group is on orthology across the tree of life, with an increased focus on microbes. 
They are responsible for the comprehensive [OMA database](https://omabrowser.org/oma/home/) together with its [ecosystem of tools](https://omabrowser.org/oma/tools/), 
and for the [Quest for Orthologs benchmark](https://questfororthologs.org/). 

The SIB members visiting us are [Natasha Glover](https://scholar.google.com/citations?hl=en&user=_TwFg1QAAAAJ)
(Associate Director, Comparative Genomics Group, Swiss Institute Bioinformatics & University of Lausanne), 
[Stefano Pascarelli](https://scholar.google.com/citations?hl=en&user=UU4EztcAAAAJ) (Postdoctoral researcher), 
[Irene Julca](https://scholar.google.com/citations?hl=en&user=ruCrGm0AAAAJ) (Senior postdoctoral researcher),
[David Moi](https://scholar.google.com/citations?hl=en&user=XgOh39oAAAAJ) (Senior postdoctoral researcher), 
[Alex Warwick Vesztrocy](https://scholar.google.co.uk/citations?user=22MWmmgAAAAJ&hl=en) (Postdoctoral researcher),
and [Mauricio Langleib](https://scholar.google.com/citations?hl=en&user=v0M39e0AAAAJ) (PhD student).

## When 
Both groups will be in the Institute between the 20th to the 24th of May, and there will be a workshop on the morning of the
21st (Tuesday) and on the afternoon of the 22nd (Wednesday). 
Tuesday's section theme will be "Comparative Genomics of the Tree of Life", and on Wednesday we'll have a "Petascale Structure Prediction and Phylogenetics" session, 
with a panel discussion led by our own [Dipali Singh](https://quadram.ac.uk/people/dipali-singh/). 
On Tuesday afternoon we will have the guests in our Data Science at QIB group meeting, where we will have an open discussion on topics brought up by the members of the group.

The list of NBI Calendar Events (no registration needed, but you can import the invites to Outlook) can be seen at:

Event | When | Where 
------|------|-------
[Workshop: Comparative Genomics of the Tree of Life](https://intranet.nbi.ac.uk/infoserv/cgi-bin/calendar/default.asp?id=66310)         | 21st May,  9:30-12:00 | UG55 ABC
[QIB Data Science group meeting](https://intranet.nbi.ac.uk/infoserv/cgi-bin/calendar/default.asp?id=66313)                             | 21st May, 14:00-16:30 | UG55 AB
[Workshop: Petascale Structure Prediction and Phylogenetics](https://intranet.nbi.ac.uk/infoserv/cgi-bin/calendar/default.asp?id=66311) | 22nd May, 14:00-17:00 | UG55 ABC

Please contact Leo if you would like to talk to the guests during their visit. 
They will be available for meetings on Monday (20th) and Thursday (23rd).


## References 
A small list of recent selected publications from the groups involved in the workshop (with guests highlighted in bold):
 
* Majidian, S., Nevers, Y., Kharrazi, A. Y., **Vesztrocy, A. W.**, **Pascarelli, S.**, **Moi, D.**, **Glover, N.**, Altenhoff, A. M., & Dessimoz, C. (2024). Orthology inference at scale with FastOMA. [_bioRxiv_ (p. 2024.01.29.577392)](https://doi.org/10.1101/2024.01.29.577392).
*  Nevers, Y., **Warwick Vesztrocy, A.**, Rossier, V., Train, C.-M., Altenhoff, A., Dessimoz, C., & **Glover, N. M.** (2024). Quality assessment of gene repertoire annotations with OMArk. [_Nature Biotechnology_](https://doi.org/10.1038/s41587-024-02147-w).
* **Moi, D.**, Dessimoz, C., Nevers, Y., **Steinegger, M.**, Bernard, C., & **Langleib, M.** (2023). Structural phylogenetics unravels the evolutionary diversification of communication systems in gram-positive bacteria and their viruses. [_bioRxiv_ (p. 2023.09.19.558401)](https://doi.org/10.1101/2023.09.19.558401).
* Nevers, Y., Jones, T. E. M., Jyothi, D., Yates, B., Ferret, M., Portell-Silva, L., Codo, L., Cosentino, S., Marcet-Houben, M., Vlasova, A., Poidevin, L., Kress, A., Hickman, M., Persson, E., Piližota, I., Guijarro-Clarke, C., OpenEBench team the Quest for Orthologs Consortium, Iwasaki, W., Lecompte, O., … Altenhoff, A. (2022). The Quest for Orthologs orthology benchmark service in 2022. [_Nucleic Acids Research_, _50_(W1), W623–W632](http://dx.doi.org/10.1093/nar/gkac330).
* Hodcroft, E. B., De Maio, N., Lanfear, R., MacCannell, D. R., Minh, B. Q., Schmidt, H. A., Stamatakis, A., Goldman, N., & Dessimoz, C. (2021). Want to track pandemic variants faster? Fix the bioinformatics bottleneck. [_Nature_, _591_(7848), 30–33](http://dx.doi.org/10.1038/d41586-021-00525-x).
* Lee, S., Kim, G., Karin, E. L., Mirdita, M., Park, S., Chikhi, R., Babaian, A., Kryshtafovych, A., & **Steinegger, M.** (2023). Petascale Homology Search for Structure Prediction. [_bioRxiv_ (p. 2023.07.10.548308)](https://doi.org/10.1101/2023.07.10.548308).
* Barrio-Hernandez, I., Yeo, J., Jänes, J., Mirdita, M., Gilchrist, C. L. M., Wein, T., Varadi, M., Velankar, S., Beltrao, P., & **Steinegger, M.** (2023). Clustering-predicted structures at the scale of the known protein universe. [_Nature_, 622, 637–645](https://doi.org/10.1038/s41586-023-06510-w)
* Kim, J., & **Steinegger, M.** (2023). Metabuli: sensitive and specific metagenomic classification via joint analysis of amino-acid and DNA. [_bioRxiv_ (p. 2023.05.31.543018)](https://doi.org/10.1101/2023.05.31.543018).
* Mirdita, M., Schütze, K., Moriwaki, Y., Heo, L., Ovchinnikov, S., & **Steinegger, M.** (2022). ColabFold: making protein folding accessible to all. [_Nature Methods_, 1–4](http://dx.doi.org/10.1038/s41592-022-01488-1).
* **Steinegger, M.**, Mirdita, M., & Söding, J. (2019). Protein-level assembly increases protein sequence recovery from metagenomic samples manyfold. [_Nature Methods_, _16_(7), 603–606](http://dx.doi.org/10.1038/s41592-019-0437-4).
